src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String BUY = "buy";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        readFromFile(fromFileName);
+        String report = createReport(getCreateDataForReport(readFromFile(fromFileName)));
+        writeFile(toFileName, report);
+    }
+
+    private String[] readFromFile(String fromFileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            StringBuilder stringBuilder = new StringBuilder();
+            String temp;
+            while ((temp = reader.readLine()) != null) {
+                stringBuilder.append(temp).append(System.lineSeparator());
+            }
+            String [] records = stringBuilder.toString().split("\\n");
+            return records;
+
+        } catch (IOException e) {
+            throw new RuntimeException("Error while reading file: " + fromFileName, e);
+        }
+    }
+
+    private int[] getCreateDataForReport(String[] records) {
+        int countBuy = 0;
+        int countSupply = 0;
+        int result;
+        for (String element : records) {
+            String[] line = element.split(",");
+            if (line[0].equals(BUY)) {
+                countBuy += Integer.parseInt(line[1]);
+            } else {
+                countSupply += Integer.parseInt(line[1]);
```

Comment:
let's create a variable for `Integer.parseInt(line[1])` and use it on both lines